ocaml/xapi/xapi_vusb.ml

```diff
@@ -23,6 +23,17 @@ let m = Mutex.create ()
 let create ~__context ~vM ~uSB_group ~other_config =
   let vusb = Ref.make () in
   let uuid = Uuid.to_string (Uuid.make_uuid ()) in
+  let attached_vusbs = Db.VM.get_VUSBs ~__context ~self:vM in
+  (* At most 6 VUSBS can be attached to one vm *)
+  if List.length attached_vusbs > 5 then
+    raise (Api_errors.Server_error(Api_errors.operation_not_allowed,
```

Comment:
Let's make this a specific error message: "TOO_MANY_VUSBS"

Reply:
updated.